### PR TITLE
feat(banner): slow website banner

### PR DESCRIPTION
**Pull request by zaccolley**
Screenshot 2022-02-24 at 13 05 09

Screenshot 2022-02-24 at 13 05 26

### CODE REVIEW

File: web/locales/en/messages.ftl [hunk 1]
```diff
@@ -13,6 +13,9 @@ email-opt-in-info-sub-with-challenge = Receive emails such as challenge and goal
 email-opt-in-privacy-v2 = By opting in to receive emails you state that you are okay with Mozilla handling this info as explained in Mozilla’s <privacyLink>Privacy Policy</privacyLink>.
 indicates-required = * Indicates required field
 not-available-abbreviation = N/A
+banner-error-slow-1 = Sorry, Common Voice is running slowly. Thanks for your interest.
+banner-error-slow-2 = We're recieving a lot of traffic and are currently investigating the issues.
```
Comment (phirework) [inline web/locales/en/messages.ftl:17]: Should be "receiving"

File: web/locales/en/messages.ftl [hunk 2]
```diff
@@ -13,6 +13,9 @@ email-opt-in-info-sub-with-challenge = Receive emails such as challenge and goal
 email-opt-in-privacy-v2 = By opting in to receive emails you state that you are okay with Mozilla handling this info as explained in Mozilla’s <privacyLink>Privacy Policy</privacyLink>.
 indicates-required = * Indicates required field
 not-available-abbreviation = N/A
+banner-error-slow-1 = Sorry, Common Voice is running slowly. Thanks for your interest.
```
Comment (phirework) [inline web/locales/en/messages.ftl:16]: Would recommend just "Sorry, Common Voice is running more slowly than usual", without the second part.

File: web/locales/ca/messages.ftl [hunk 3]
```diff
@@ -14,6 +14,9 @@ email-opt-in-info-sub-with-challenge = Rebreu correus com ara recordatoris d'obj
 email-opt-in-privacy-v2 = En optar per rebre correus electrònics, accepteu que Mozilla gestioni aquesta informació en els termes descrits en la seva <privacyLink>política de privadesa</privacyLink>.
 indicates-required = * Indica un camp necessari
 not-available-abbreviation = N/D
+banner-error-slow-1 = Ens sap greu, el Common Voice està tardant a carregar-se, gràcies per l'interès.
+banner-error-slow-2 = La plataforma va molt carregada últimament, estem investigant els problemes i esperem que torne prompte.
+banner-error-slow-link = Estat del projecte
```
Comment (phirework) [inline web/locales/ca/messages.ftl:20]: Anything in a locale other than English is going to get overwritten next time there's a sync - I don't think that's an issue with this specific deploy but just a heads up

**Review by phirework — CHANGES_REQUESTED** [review]
A typo and a couple of comments

File: web/locales/en/messages.ftl (hunk 2 above)
Comment (zaccolley) [reply web/locales/en/messages.ftl:16]: This is good feedback but I will leave this for now, I added 'Thanks for your interest' based on Hillary's feedback and we could go back and forth on the copy forever.

File: web/locales/en/messages.ftl (hunk 1 above)
Comment (zaccolley) [reply web/locales/en/messages.ftl]: Thank you!

**Review by phirework — APPROVED** [review]

**Pull request merged**